spec/fixtures/site_settings/locale_override.yml

```diff
@@ -0,0 +1,14 @@
+basic:
+  default_locale:
+    default: 'en'
+    enum: 'LocaleSiteSetting'
+
+search:
+  min_search_term_length:
+    client: true
+    default:
+      development: 1
```

Comment:
I prefer the syntax: 

```
default: 'test'
locale_default: 
   zh_CN: 1
```

It is a bit more verbose but I feel it is a lot clearer.